src/views/registration-js/content.php

```diff
@@ -167,7 +172,7 @@ class="tribe-tickets__item__attendee__fields__form <?php echo sanitize_html_clas
 						</div>
 					</div>
 				</div>
-				<?php $this->template( 'registration-js/attendees/content', array( 'event_id' => $event_id, 'tickets' => $tickets, 'provider' => $providers[0] ) ); ?>
+				<?php $this->template( 'registration-js/attendees/content', [ 'event_id' => $event_id, 'tickets' => $tickets, 'provider' => $providers[0] ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 3fcc87720275c82e706e0acced1fee85</sub>